.gitlab-ci.yml

```diff
@@ -59,6 +59,7 @@ init_workspace:
   tags:
     - mender-qa-slave
   script:
+    - export
```

Comment:
I see your intention.

But I didn't do this originally because I will print secrets like `SSH_PRIVATE_KEY` and `GITHUB_BOT_PASSWORD`. These variables have special characters and GitLab cannot mask them.

Reply:
I suggest this then:

```
export | egrep -v 'KEY|SECRET|PASSWORD'
```

It matches all the credentials we currently have.

Reply:
Hmm, this doesn't work apparently, because some keys are multiline and hence most of the content escapes this filter. I'll try to base it on the variable list instead, since that is ultimately what is important.

Reply:
I just removed this for now, to get the rest of the PR in, but I will probably return to it soon.